style.css

```diff
@@ -81,7 +80,11 @@ li {
   list-style-type: none;
 }
 
-.social-media > li img {
+.social-media > a > li > img {
```

Comment:
- [ ] As stated the project requirement and suggested by the previous reviewer You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly reduce the size of these icons to match the figma design, The Figma design set the width and the height to `20px` Kindly apply the same values to your icons.

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205245645-7223db63-1073-40f4-b82d-c0eb585274b8.PNG)

Your Design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/205245918-2079ac66-8b73-4939-9789-05f9c05f1041.PNG)